src/config.ts

```diff
@@ -60,6 +60,27 @@ class Config {
     };
   }
 
+  static get matomoConfiguration() {
+    const matomoUrlBase = this.getEnvOrError(
```

Comment:
If this is an optional config, we don't need to error. E.g. if the application can function without it, we do not need to error.

Without some API configs, the application will be broken and useless.

Reply:
Ill check if it is optional config

Reply:
Matomo config seems to require URL. Do you think we should add some logic to not include Matomo at all if for example the URL enrivonment variables is missing?

<img width="1054" alt="Screenshot 2022-02-01 at 12 18 37" src="https://user-images.githubusercontent.com/15219142/151951040-3949ed22-b57d-45f4-bea0-529026543287.png">

Reply:
Maybe the simplest solution would be to just have the Matomo enabled if all three env variables are defined. Not sure if it is even necessary to have any of those variables other than  `NEXT_PUBLIC_MATOMO_ENABLED` in environment variables

Reply:
I'm fine with configuring each environment with the necessary variables and then using matomo's configs to ensure that we don't push stats from other than production environments.

I added the original comment because the CI is breaking on missing env variables. If we should always provide them, then we should configure the CI env to have them.

I mean it'd be ideal if it weren't included when there are no configurations so that we can ensure we don't accidentally log something from an environment that's irrelevant. However, as I recall that's difficult and matomo doesn't provide an easy way to do it out of the box.